theme/less/_homepage.less

```diff
@@ -1,23 +1,33 @@
-.homepage
+.homepage div
 {
   grid-template-areas: 
     "header header"
     "main main"
     "footer footer";
 }
 
-.homepage main article
+.homepage div main article
 {
   display: grid;
-  grid-template-columns: repeat(auto-fill, minmax(346px, 1fr));
+}
+
+.homepage div main article h2
+{
+  grid-column: 1 e("/") 3;
```

Comment:
This modification solves the heading problem also takes of risks related to exact sizings.

Reply:
What is `e`? I couldn't find anything about e in https://developer.mozilla.org/en-US/docs/Web/CSS/grid-column.

Can you please describe to me this implementation please?

Also I don't understand how this style affects siblings of this selector, I can see that this act as merge for top column of 2 column grid, but I don't understand how and where are we specifying that the grid is 2 column?

Reply:
Here you can play around with this this property. 
https://codepen.io/pen/?&editable=true=https%3A%2F%2Fdeveloper.mozilla.org%2Fen-US%2Fdocs%2FWeb%2FCSS%2FCSS_Grid_Layout%2FAuto-placement_in_CSS_Grid_Layout

Also here are more examples.
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Auto-placement_in_CSS_Grid_Layout

Reply:
> It escapes the "/" sign as LESS processes it as "divider" sign.

Ohh I see, thanks for explanation, sounds like LESS is stuck in his his old implementation. It's sad that we need to escape :(

> Here you can play around with this this property.

This codepen is empty.

> Here 1 / 3 means starting from 1st ending to 3rd column

This yet sounds contr-intuituve to me that specifying `grid-column` on wrapper's first child, sets parent element to 2 column grid, this sounds bit too magical for me, I think they are imitating bootstrap, when you also don't specify parent to grid explicitly AFAIK. Also I couldn't find an example where grid structure is not defined on the wrapper on MDN docs. I'm also bit hesitant, for example if we add 3-rd box below we have to adjust first child and track the grid amount on the children properties rather than on the wrapper which will make things more understandable, for person not familiar with this?

I think if we develop a grid model on CSS we may have more self explanatory implementation, I'm thinking about something like:
```
// 2 stands for 2 column grid on mid sizes?
.grid-md-2 {
  display: grid;
  grid-template-columns: auto auto; // or: 1fr 1fr?
}

// md stands for make 2 columns on medium size, similar to https://getbootstrap.com/docs/4.0/layout/grid/ 
.col-md-2 {
    grid-column: 1 e("/") 3;
}
```

I think this way we can reuse implementation also on the main layouting and introduce 3 or 4 column grid when/if needed. As we only need 2 column grid I think it's fine, this implementaion has it's own cons, for example if we decide to introduce more than 2 boxes, let's say 4, then we need to create 4 column grid and adjust parents which we could have avoided by wrapper, but I think for now this is fine, unless we feel the need to restructure.

Also current implementation will allow us to be more flexible regarding the content of the homepage, ex.: if we use `<h1>` instead of `<h2>` we don't need to adjust styles, or if we use `<ul>` instead of `<ol>`.

Please let me know what you think @naarrek especially I wonder if it's only me for whom not specifying the grid model on parent item feels too magical. Also feel free to find different naming or maybe you can think of a better implementation, as you are more advanced than me on this already :)

Reply:
Also I wonder if we need `md`, considering we have only 1 size. Also hesitant about that, will wait on your thoughts though.